### PR TITLE
feat: first api implementation

### PR DESCRIPTION
**Pull request by pietro-tota**
#### List of Changes
Search wallet api implementation

#### Motivation and Context
Search wallet api implementation

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: cmd/app/main.go [hunk 1]
```diff
@@ -1,47 +1,22 @@
...
-			log.Printf("wallet user id: [%s]", wallet.UserID)
-			log.Printf("wallet detail type: [%s]", wallet.Details.Type)
-			if len(wallet.Applications) > 0 {
-				log.Printf("wallet application ID: [%s]", wallet.Applications[0].ID)
-				log.Printf("wallet application status: [%s]", wallet.Applications[0].Status)
-			}
-		}
-	}
-
+	cosmosdb, _ := DB.GetMongoClient()
```
Comment (giovanniberti) [inline cmd/app/main.go:18]: No check for error? :)
```suggestion
	cosmosdb, err := DB.GetMongoClient()
	if err != nil {
	    log.Fatalf("Unable to create Mongo client");
	}
```

**Review by giovanniberti — APPROVED** [review]
Good job! 🚀

File: cmd/app/main.go (hunk 1 above)
Comment (pietro-tota) [reply cmd/app/main.go]: fixed with https://github.com/pagopa/pagopa-payment-wallet-helpdesk-service/pull/8/commits/1cbde9a29295174d7259e53597ad44815d13141f

**Review by giovanniberti — APPROVED** [review]

**Review by petretiandrea — DISMISSED** [review]
🚀

**Review by giovanniberti — APPROVED** [review]

**Pull request merged**